src/gemm_v2.cpp

```diff
@@ -359,12 +393,7 @@ miopenStatus_t CallGemm(const Handle& handle,
 
 // do row-to-column major conversion here
 #if MIOPEN_USE_MIOPENTENSILE
-    if((gemm_desc.isColMajor && gemm_desc.dataType == miopenFloat)
-#if MIOPEN_USE_ROCBLAS
-       ||
-       (!gemm_desc.isColMajor && gemm_desc.dataType != miopenFloat)
-#endif
-           )
+    if(gemm_desc.isColMajor)
```

Comment:
why the logic for swapping row and col is different depending on whether MIOpenTensile is used or now

Reply:
AFAIK it was different with MIOpen's gemm definition when this PR was created. We adopt a logic that fits tensile's definition.

Reply:
Please add a comment here (and other places which have this swapping logic ) explaining what is the difference between MIOpen's GEMM definition Tensil's definition of "col-major". That would be our future reference if we need to unify the definition

Reply:
will add comment in skip CI mode